src/ContextFreeTasks.Shared/ContextFreeTask_T.cs

```diff
@@ -9,7 +9,9 @@ public struct ContextFreeTask<T>
     {
         private readonly Task<T> _task;
         public Task<T> Task => _task ?? FromResult(default(T));
+        public T Result => GetAwaiter().GetResult();
```

Comment:
This is uneasy. For consistency with `ValueTask`, the `Result` should be implemented like this. However, I don't prefer blocking wait.
If breaking change is allowed, I really want the `Task.Result` to throw an exception instead of blocking wait when it has not completed yet.